flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/sink/JdbcSink.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.jdbc.sink;
+
+import org.apache.flink.annotation.Internal;
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.api.connector.sink2.Committer;
+import org.apache.flink.api.connector.sink2.StatefulSink;
+import org.apache.flink.api.connector.sink2.TwoPhaseCommittingSink;
+import org.apache.flink.connector.base.DeliveryGuarantee;
+import org.apache.flink.connector.jdbc.JdbcExactlyOnceOptions;
+import org.apache.flink.connector.jdbc.JdbcExecutionOptions;
+import org.apache.flink.connector.jdbc.datasource.connections.JdbcConnectionProvider;
+import org.apache.flink.connector.jdbc.datasource.statements.JdbcQueryStatement;
+import org.apache.flink.connector.jdbc.internal.JdbcOutputSerializer;
+import org.apache.flink.connector.jdbc.sink.committer.JdbcCommitable;
+import org.apache.flink.connector.jdbc.sink.committer.JdbcCommitableSerializer;
+import org.apache.flink.connector.jdbc.sink.committer.JdbcCommitter;
+import org.apache.flink.connector.jdbc.sink.writer.JdbcWriter;
+import org.apache.flink.connector.jdbc.sink.writer.JdbcWriterState;
+import org.apache.flink.connector.jdbc.sink.writer.JdbcWriterStateSerializer;
+import org.apache.flink.core.io.SimpleVersionedSerializer;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+
+/**
+ * Flink Sink to produce data into a jdbc database.
+ *
+ * @see JdbcSinkBuilder on how to construct a JdbcSink
+ */
+@PublicEvolving
```

Comment:
In general, we use the `Experimental` annotation instead of `PublicEvolving`  or `Public`  for new features or new apis. But `JdbcSink` was marked as `PublicEvolving`  in the first version.

It's better to remark it to `Experimental` before releasing, right?

Reply:
uh. maybe not